examples/visualization/plot_ssp_projs_topomaps.py

```diff
@@ -19,11 +19,13 @@
 
 data_path = sample.data_path()
 
-ecg_fname = data_path + '/MEG/sample/sample_audvis_ecg_proj.fif'
+ecg_fname = data_path + '/MEG/sample/sample_audvis_ecg-proj.fif'
 ave_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
 
-evoked = read_evokeds(ave_fname, condition='Left Auditory')
-projs = read_proj(ecg_fname)
+evoked = read_evokeds(ave_fname, condition='Left Auditory',
+                      baseline=(None, 0.))
+# Omit the first 4 projectors as they are already included in the evoked data.
+projs = read_proj(ecg_fname)[4:]
```

Comment:
I wonder why the PCA and EEG projectors are included in the EOG and ECG projection files. Maybe they should be removed.